inst/docker/minimal/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM rocker/r-base:latest
+
```

Comment:
Just want to double check here - using a latest tag is usually a big dangerous because it's a moving target. Would it make sense to build from a specific version instead, to be updated when you are absolutely ready?